firebase/headers.js

```diff
@@ -109,6 +111,7 @@ module.exports = [
 
             // HubSpot chat
             'https://forms.hsforms.com',
+            'https://track.hubspot.com',
```

Comment:
That document was fairly recently published, I don't know that the folks that created the CSP rules were referencing it. Or, if they were referencing it, they decided to make the rules more restrictive. The document seems overly broad to me, potentially because it needs to work for every HS service and every HS customer.

Ultimately, I can only really speak to the one entry I added. We had one CSP error from a tracking pixel at `track.hubspot.com`. I saw that most entries here use a specific subdomain. There are cases where a wildcard is required, like when requests use a seemingly-random subdomain or one of an unknown variety of regional subdomains. In this case, it didn't seem like allowing _all_ subdomains was required, so I opted for the more restrictive option of using the specific subdomain.

Reply:
When I was adding the HubSpot chat to things, I intentionally used a much more restrictive CSP than what they recommended.